src/glossary.rst

```diff
@@ -2231,6 +2231,17 @@ function item type
 A :dt:`function item type` is a unique anonymous :t:`function type` that
 identifies a :t:`function`.
 
+.. _fls_WMaE58yv1joW:
+
+function lifetime elision
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:dp:`fls_tZMmRHua1S8K`
+:dt:`Function lifetime elision` is a form of :t:`lifetime elision` that applies
+to :t:`[function]s`, :t:`function pointer type parameter` and :t:`[path]s`
```

Comment:
I just noticed this, function pointer type parameter**s** (plural, since everything else is plural).